Lib/json/tool.py

```diff
@@ -25,24 +25,35 @@ def main():
                         help='a JSON file to be validated or pretty-printed')
     parser.add_argument('outfile', nargs='?', type=argparse.FileType('w'),
                         help='write the output of infile to outfile')
+    group = parser.add_mutually_exclusive_group()
+    group.add_argument('--indent', default=4, type=int,
+                       help='Indent level (number of spaces) for '
+                       'pretty-printing. Defaults to 4.')
+    group.add_argument('--no-indent', action='store_const', dest='indent',
```

Comment:
@haypo the following illustrates the three options

```python
import json
obj = [1, 2]

print('--indent=4')
print(json.dumps(obj, indent=4))

print('--no-indent')
print(json.dumps(obj, indent=None))

print('--compact')
print(json.dumps(obj, separators=(',', ':')))
```

which produces the following output

```
--indent=4
[
    1,
    2
]
--no-indent
[1, 2]
--compact
[1,2]
```